Removing login-us.microsoftonline.com

BernieEllis opened the pull request:
login.microsoftonline.us is the new AAD endpoint for Azure Government. login-us.microsoftonline.com is being deprecated.

tysonn commented:
Reviewing this PR.

Thraka commented:
@tysonn  #sign-off

Thraka commented:
#hold-off I take that back, a few links need fixing. Working in email with the author

Thraka commented:
@tysonn Quick scan and things look good to merge in...

BernieEllis committed:
Removing login-us.microsoftonline.com

login.microsoftonline.us is the new AAD endpoint for Azure Government. login-us.microsoftonline.com is being deprecated.
